### ISSUE TITLE
Eclipse Fails To Start

### ISSUE THREAD

**Issue by jeromebridge** [life-ops/devops#2]
```
access control disabled, clients can connect from any host** (eclipse:26): 
WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OmXaq4gfr1: Connection refused** (java:27): 
WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OmXaq4gfr1: Connection refused
(java:27): Gdk-ERROR **: The program 'java' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadAccess (attempt to access private resource denied)'.
 (Details: serial 204 error_code 10 request_code 130 (MIT-SHM) minor_code 1)
 (Note to programmers: normally, X errors are reported asynchronously;
  that is, you will receive the error a while after causing it.
  To debug your program, run it with the GDK_SYNCHRONIZE environment
  variable to change this behavior. You can then get a meaningful
  backtrace from your debugger if you break on the gdk_x_error() function.
```

**Comment by jeromebridge** [comment]
Could be related: https://github.com/osrf/docker_images/issues/21

**Comment by jeromebridge** [comment]
Closing...hasn't happened again for some time.
